CVE-2016-1897/cvex.yml

```diff
@@ -0,0 +1,10 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+  command:
+    - "php -S 0.0.0.0:9999 -f /var/www/html"
+ubuntu2:
+  playbook: ubuntu2.yml
+  command:
+    - "/var/www/generate_data.sh %ubuntu1%"
+    - "cd /var && docker compose up -d --build && docker-compose exec web ffmpeg -i /var/www/html/test.m3u8 /var/www/html/6215ca35009ea.mp4 "
```

Comment:
Please move this command into the Ansible playbook. It'd be great to have a snapshot that does already have this container up and running. Remember that Ansible supports Docker commands, no need to use the shell commands.